server/sonar-web/src/main/js/components/issue/components/IssueCommentLine.js

```diff
@@ -85,10 +85,10 @@ export default class IssueCommentLine extends React.PureComponent {
           <Avatar
             className="little-spacer-right"
             hash={comment.authorAvatar}
-            name={comment.authorName}
+            name={comment.authorName || comment.author}
```

Comment:
Maybe this should be done to the "title" attribute of the parent div. If so, we would have 3 repetitions of the same code so we might need to do `const name = comment.authorName || comment.author` first